README.md

```diff
@@ -27,8 +27,9 @@ $ yarn add injection-js
 > For ES5 `Class` syntax and TypeScript you need a polyfill for the [Reflect API](http://www.ecma-international.org/ecma-262/6.0/#sec-reflection).
 > You can use:
 >
+> - [reflection](https://www.npmjs.com/package/@abraham/reflection) 👉 only 3kb (recommended)
```

Comment:
I saw this in the mailing list yesterday. Maybe too early to be recommended? :-)

Reply:
I'm already using it so all good 😎 . I've talked with Abraham about but via GitHub issues as well. But no biggie I can remove word "recommended" 😉